src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -330,12 +338,22 @@ class FluxPipeline : public DiffusionPipeline {
 
             auto scheduler_step_result = m_scheduler->step(noise_pred_tensor, latents, inference_step, generation_config.generator);
             latents = scheduler_step_result["latent"];
+
+            if (do_callback) {
+                if (callback(inference_step, latents)) {
+                    return ov::Tensor(ov::element::u8, {});
+                }
+            }
         }
 
         latents = unpack_latents(latents, generation_config.height, generation_config.width, vae_scale_factor);
         return m_vae->decode(latents);
     }
 
+    ov::Tensor decode(const ov::Tensor latent) override {
```

Comment:
looks like it should also perform `unpack_latents` ?

Reply:
https://github.com/openvinotoolkit/openvino.genai/pull/1237